distros/kubernetes/nvsentinel/charts/csp-health-monitor/templates/deployment.yaml

```diff
@@ -36,17 +36,54 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "csp-health-monitor.fullname" . }}
+      {{- if and .Values.global.datastore (eq .Values.global.datastore.provider "postgresql") }}
+      initContainers:
+        - name: fix-cert-permissions
+          image: "docker.io/bitnamilegacy/os-shell:12-debian-12-r30"
```

Comment:
Can we make the image name a value? Some user may want to use their custom mirror images to avoid docker rate limits